mos-platform/atari5200-super/link.ld

```diff
@@ -0,0 +1,278 @@
+/* Atari 5200 linker script.
+
+  32 KiB banks, 1 bank would be a normal 32 KiB cartridge, 128 KiB
+  through 512 KiB would be a Super Cart.
+
+  We have a common "fixed" region that is a prefix for all banks.  The
+  last bank has a tail used to initialize the ROM.
+
+  https://github.com/atari800/atari800/blob/ATARI800_5_0_0/DOC/cart.txt
+*/
+
+/* Default cartridge size.  This isn't the largest possible size to
```

Comment:
In the absence of other concerns, this should be the largest size, and tests should be provided for any other sizes that might break for independent reasons.

Reply:
Will do.